src/strings/unicode_ops.c

```diff
@@ -1,6 +1,17 @@
 /* Compares two strings, returning -1, 0 or 1 to indicate less than,
  * equal or greater than based on Unicode Collation Algorithm */
-MVMint64 MVM_unicode_string_compare(MVMThreadContext *tc, MVMString *a, MVMString *b) {
```

Comment:
Think the comment above wants changing to reflect the new range of return values.